src/benchmarks/gc/src/mypy.ini

```diff
@@ -0,0 +1,5 @@
+[mypy]
+python_version = 3.7
```

Comment:
so why are we limiting this to 3.7? does this mean it would complain if I'm running on 3.8?

Reply:
No, it means to use at least 3.7 version. This is in accordance with the original docs and tests made by Andy and specified in the README. This file is only to add `mypy` exceptions (this should be kept to a minimum), as `mypy` sometimes fails in linting. This is equivalent to the pylintrc file Andy added for some pylint exceptions.

Reply:
I see; when I looked up the doc for this it said this is the version, not the oldest version. that's fine then.